### ISSUE TITLE
not placing soul shards into soul pouch

### ISSUE THREAD

**Issue by vampyricus** [arkayenro/arkinventory-classic#12]
When using a soul pouch ark is not placing all the soul shards ONLY into the soul pouch and custom rule for that is not working

**Comment by arkayenro** [comment]
do you mean its trying to put other things in there as well, or its not moving soul shards into them?

rules are for visual positioning only, they dont apply to what goes in a bag

**Comment by vampyricus** [comment]
it's moving the soulshards out of the shard pouch and into other bags and then its also trying to place other items INTO the shard bag

**Comment by arkayenro** [comment]
it shouldnt be pulling them out unless the underlying blizzard data for the soul shards and the bags are wrong

i dont have a soul bag at this point to be able to test it with so it might take a while to sort out

**Comment by vampyricus** [comment]
i noticed in a previous post about another bag issue that there's a script that can pull the bag ID ... if i get that would that help u be able to check it? 

if necessary it can wait though.

**Comment by arkayenro** [comment]
not really, it sounds like something is weird so i'll probably need to b able to test it myself to figure t out.

warlock is at 10 though, just need to get a soul bag

**Comment by arkayenro** [comment]
can you try r848-alpha?  i've got herbs and arrows to work properly so im hoping shards work as well

looks like the itemfamily value is different in classic
